gef.py

```diff
@@ -6404,6 +6404,46 @@ def do_invoke(self, argv):
         gef_print(chunk.psprint())
         return
 
+@register_command
+class GlibcHeapChunkRangeCommand(GenericCommand):
+    """Display information on all heap chunks within memory region {from, to}.
+    First address should be valid chunk address while the last one could be arbitrary > first.
+    See https://github.com/sploitfun/lsploits/blob/master/glibc/malloc/malloc.c#L1123."""
+
+    _cmdline_ = "heap chunkrange"
+    _syntax_  = "{:s} LOCATION_FROM LOCATION_TO".format(_cmdline_)
+    _example_ = "\n{0:s}\n{0:s} 0x4000aa00 0x4000da00".format(_cmdline_)
+
+    def __init__(self):
+        super(GlibcHeapChunkRangeCommand, self).__init__(complete=gdb.COMPLETE_LOCATION)
+        return
+
+    @only_if_gdb_running
+    def do_invoke(self, argv):
+        if len(argv) != 2:
+            err("Missing range addresses")
+            self.usage()
+            return
+        
+        AddrFrom = to_unsigned_long(gdb.parse_and_eval(argv[0]))
+        AddrTo = to_unsigned_long(gdb.parse_and_eval(argv[1]))
+
+        if (AddrTo <= AddrFrom):
+            err("LOCATION_TO <= LOCATION_FROM")
```

Comment:
Error message could be more descriptive. e.g. `LOCATION_TO`/`END_LOCATION` must be an address greater than `LOCATION_FROM`/`START_LOCATION`.